packages/block-library/src/embed/embed-preview.native.js

```diff
@@ -53,19 +59,64 @@ const EmbedPreview = ( {
 		}
 	}
 
+	const { scripts, provider_url: providerUrl } = preview;
+	const html = 'photo' === type ? getPhotoHtml( preview ) : preview.html;
+	const parsedHost = new URL( url ).host.split( '.' );
+	const parsedHostBaseUrl = parsedHost
+		.splice( parsedHost.length - 2, parsedHost.length - 1 )
+		.join( '.' );
+	const iframeTitle = sprintf(
+		// translators: %s: host providing embed content e.g: www.youtube.com
+		__( 'Embedded content from %s' ),
+		parsedHostBaseUrl
+	);
+
+	const embedWrapper =
+		/* We should render here: <WpEmbedPreview html={ html } /> */
+		'wp-embed' === type ? null : (
```

Comment:
We'll implement the WP embed preview component when tackling [this task](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3744).